2-mandatory/4-stocks.js

```diff
@@ -34,8 +34,20 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  let newArr = [];
+  for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+    let avgStockPrice = 0;
+    for (let x = 0; x < closingPricesForAllStocks[i].length; x++) {
+      avgStockPrice += closingPricesForAllStocks[i][x];
+    }
+    newArr.push(
+      +(avgStockPrice / closingPricesForAllStocks[i].length).toFixed(2)
```

Comment:
Um... it works as number of stock (i) equals to number of days (x). So, it be incorrect when we only calculate 4 days of stocks prices.